index.js

```diff
@@ -389,7 +486,9 @@ function createQuestionnaireTemplateHelper({
                 ensureRouteTargetsHaveCorrespondingState(),
                 ensureAllConditionDataReferencesHaveCorrespondingQuestion(),
                 ensureSectionSchemasAreValid(),
-                ensureAllSectionsHaveThemes()
+                ensureAllSectionsHaveThemes(),
+                ensureAllRoutesAreOnOwnMachine(),
+                ensureAllSectionsAreOwnedByOneTask()
```

Comment:
could we be really specific here and call these something like:

ensureAllRoutesReferencePageIdsOnSameMachine
ensureAllSectionsAreOwnedByOneTaskOnly